src/CodegenClassish.hack

```diff
@@ -380,7 +411,7 @@ abstract class CodegenClassish implements ICodeBuilderRenderer {
     $generated_from =
       $this->generatedFrom ? $this->generatedFrom->render() : null;
 
-    $doc_block_parts = \array_filter(varray[$this->docBlock, $generated_from]);
+    $doc_block_parts = Vec\filter_nulls(vec[$this->docBlock, $generated_from]);
```

Comment:
This behaves differently when either string is `''` or `0`. This is a breaking change, but it is very very minor. Can revert if need be.

Reply:
I rewrote this to stop relying on _typechecker magic_.
This kind of _magic_ always catches me off guard.
I don't know if this is also surprising to others.
I am just being explicit for the reason of being explicit.
The behavior change is a wash `0` ✔️ `''` ❌.
```
/*
 * Calls to array_filter are rewritten depending on the type
 * of argument to have one of the following signatures:
 *
 * function(array, ?(function(Tv):bool)): array
 * function(KeyedContainer<Tk, Tv>, ?(function(Tv):bool)): array<Tk, Tv>
 * function(Container<Tv>, ?(function(Tv):bool)): array<arraykey, Tv>
 *
 * Single argument calls additionally remove nullability of Tv, i.e.:
 *
 * function(Container<?Tv>): array<arraykey, Tv>
 *
 */
 ```